### ISSUE TITLE
Unable to add new members to team

### ISSUE THREAD

**Issue by canadaduane** [go-vikunja/frontend#86]
### Description

When trying to add a member to the team, the drop-down does not autosuggest anyone, and therefore I cannot add anyone to the team.

I can confirm that the request is being made--in the network tab the `/api/v1/users?s=[query]` call is being made, but the response is always null, even when I type the initial letters of a team member that I know is in the system (for example, when I removed "ava"  from the team, then tried to add her back, typing "a" or "av" or "ava" does not autosuggest her user).

### Vikunja Frontend Version

0.20.0

### Vikunja API Version

0.20.0

### Browser and version

Firefox 106.0.1

### Can you reproduce the bug on the Vikunja demo site?

Yes

### Screenshots

![image](https://user-images.githubusercontent.com/129/199253539-102b53ed-195b-4230-8843-ee33f11c37ab.png)

**Comment by dpschen** [comment]
This is definitely a mistake. Will investigate.

**Comment by kolaente** [comment]
There have been changes to the way user Search works in the last release to avoid enumerating all users on a server. This mostly means you'll need to enter the exact username of the user you're trying to add or (if they have it enabled) the exact email or display name of the user.
Can you check if it works if you try it like that?

**Comment by canadaduane** [comment]
Yes, it works if I add the exact username. Is there a list of users somewhere?

**Comment by kolaente** [comment]
You can use the [`vikunja user list`](https://vikunja.io/docs/cli/#user-list) command to show a list of all users.

**Comment by dpschen** [comment]
Maybe we should change this so that admins can have access to the user names.
They are admins after all

**Comment by kolaente** [comment]
@dpschen The admin permission is only applied to the current team. The search is globally, on the server.

**Comment by dpschen** [comment]
Okay, that makes sense.
We should still improve the UX here somehow. And if it's only to describe what you can search for, why it's that restricted and how you can change that.

**Comment by dpschen** [comment]
Does adding via mail always work? Because that should. If the person is not registered I guess an invitation should be sent

**Comment by kolaente** [comment]
> Does adding via mail always work?

It only works if the user is discoverable by searching for the mail address.

**Comment by kolaente** [comment]
> If the person is not registered I guess an invitation should be sent

I think there's an item somewhere in the backlog about this.
